interview-questions.md

```diff
@@ -15,43 +15,58 @@ Researched answer:
 1. What is git? What is the difference between git and Github?
```

Comment:
When answering these interview questions, please remember this is your practice for interviews. Though your answers are correct. They need a more content. Research and find a definition, ensure you paraphrase and use your own words. Next week, let's focus on complete sentences that address the definition of the topic and something new that you found out.